mongodbatlas/project_invitations.go

```diff
@@ -124,8 +124,6 @@ func (s *ProjectsServiceOp) UpdateInvitationByID(ctx context.Context, groupID, i
 		return nil, nil, NewArgError("invitationID", "must be set")
 	}
 
-	invitation.ID = invitationID
-
```

Comment:
[q] If the parameters must be blank, should we instead check if they're set and reject on the call?

Reply:
The parameter `ID` inside the struct `invitation` (request body) must be black not the parameter itself. Here I was erroneously setting this param inside the body instead of using it just as a path parameter. See [Update One Project Invitation by Invitation ID](https://docs.atlas.mongodb.com/reference/api/project-update-one-invitation-by-id/).
To summarise, we are not ignoring the `ID` but we are just removing it from the request body and using it only in the path ([here I am using invitationID](https://github.com/mongodb/go-client-mongodb-atlas/blob/7103b52018f4586ce9fd620d0f99e5cc720207da/mongodbatlas/project_invitations.go#L159))